deployment/configCmd.py

```diff
@@ -39,6 +39,7 @@ def generate_configuration(quick_start_config_file, configuration_directory, for
             machine["password"] = quick_start_config_raw["ssh-password"]
         else:
             machine["ssh-keyfile-path"] = quick_start_config_raw["ssh-keyfile-path"]
+            machine["keyfile-path"] = quick_start_config_raw["ssh-keyfile-path"]
```

Comment:
I think the value of ```machine["ssh-keyfile-path"]``` is same as ```machine["ssh-keyfile-path"]```

Reply:
That's right. I removed the 'ssh-keyfile-path' setting. 
I was not sure it would be used somewhere else in the Code. The scripts still work after removing the setting. So 'ssh-keyfile-path' seems to be not used at all.